contribs/gmf/src/controllers/abstract.js

```diff
@@ -81,6 +84,9 @@ gmf.AbstractController = function(config, $scope, $injector) {
    */
   this.queryActive = true;
 
+  // Not used in this file but the QueryManager must be injected to be watched.
+  $injector.get('gmfQueryManager');
```

Comment:
Not really. If it's not injected, there is nothing... literally nothing. I you inject it, the value can be watched and something can happens. but it not like a test...